synapse/tests/test_tools_snapshot.py

```diff
@@ -0,0 +1,32 @@
+from unittest import mock
+
+import synapse.tests.utils as s_t_utils
+
+import synapse.tools.snapshot as s_tools_snapshot
+
+class PromoteToolTest(s_t_utils.SynTest):
+
+    async def test_tool_snapshot(self):
```

Comment:
You could probably test that the pausing and copying actually works by copying core.dirn to a new directory and starting a new cortex with that dir.

Reply:
Without any edits going on, you could very likely copy and start the Cortex even without the freeze/resume so i'm not sure a test like that is worthwhile. We can do a quick goback :+1:

Reply:
You would have to do a raw copy with something like `shutil` ( which could be fast but copying all the sparse slabs won't be quick ); since we cannot execute a Cell backup while Cell is frozen.